csfix

```diff
@@ -0,0 +1,6 @@
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+
+cd $SCRIPTPATH/src
+
+vendor/bin/php-cs-fixer fix ./src --rules=@Symfony --verbose --show-progress=estimating --using-cache=no
```

Comment:
ak `csfix` presunieme do `src` tak toto prestane fungovat. `vendor` staci nahradit za `./vendor` a ide to. (ale mozno mas lepsie riesenie)

Reply:
Divim sa ze to po presune nezapinda uz pri tom `cd` prikaze.